bugbug/repository.py

```diff
@@ -55,6 +55,7 @@ def _transform(commit):
         'components': list(),
         'author_experience': author_experience[commit],
         'author_experience_90_days': author_experience_90_days[commit],
+        'author_email': commit.author_email.decond('utf-8'),
```

Comment:
Did you try to run the script? `decond` must be wrong :P

Reply:
;_; Done